hastexo_guacamole_client/guacamole/client.py

```diff
@@ -0,0 +1,221 @@
+"""
+The underlying work for this code is published under the MIT License (MIT)
+(https://pypi.org/project/pyguacamole/)
+"""
+
+import socket
+import logging
+
+from .exceptions import GuacamoleError
+
+from .instruction import INST_TERM
+from .instruction import GuacamoleInstruction as Instruction
+
+# supported protocols
+PROTOCOLS = ('vnc', 'rdp', 'ssh')
+
+PROTOCOL_NAME = 'guacamole'
+
+BUF_LEN = 4096
+
+guac_logger = logging.getLogger(__name__)
+guac_logger.setLevel(logging.INFO)
+guac_logger.handlers = [logging.StreamHandler()]
+
+
+class GuacamoleClient(object):
+    """Guacamole Client class."""
+
+    def __init__(self, host, port, timeout=20, loglevel=logging.DEBUG, logger=None): # noqa E501
```

Comment:
Rather than adding the `# noqa E501` comment, I usually prefer to put

```python
     def __init__(self,
                  host,
                  port,
                  timeout=20,
                  loglevel=logging.DEBUG,
                  logger=None):
```

... because not only does this make flake8 happy, it's also more readable.

Reply:
Fixed in ea3cebaa11390a4b50b70df8cec8e09efc01283b.